icebergs_io.F90

```diff
@@ -201,7 +201,7 @@ subroutine write_restart(bergs)
    allocate(id_ij(nbergs))
 
 
-  call get_instance_filename("icebergs.res.nc", filename)
```

Comment:
This call causes the _ensNN appending to the filename. But fms_io also appends another _ensNN to the filename.